docs/RFCS/20220524_sso_authentication.md

```diff
@@ -0,0 +1,145 @@
+- Feature Name: SSO Authentication
+- Status: draft
+- Start Date: 2022-05-24
+- Authors: Cameron Nunez
+- RFC PR: #82540
+
+# Summary
+
+This RFC proposes introducing Console-issued accept tokens as a CRDB login 
+method. This change would enable single sign-on (SSO) for CRDB, allowing users
+to login automatically and have a more consolidated login experience for CRDB and 
+CC Console. 
+
+This RFC seeks to achieve the following goals:
+* Customers are able to use short-lived credentials to access CRDB.
+* Users are able to create sessions without using "something you know" 
+authentication.
+* Users, having only entered login credentials into their identity provider, 
+are able to create an active CRDB session with a Console cluster.
+* Service accounts, hosted in CC or in the customer's identity provider, are 
+able to create a CRDB session. 
+
+The following are out of scope:
+* Authorization e.g. user permissions propagation from CC to CRDB
+* Just-in-time provisioning of SQL users at login time
+* SSO for DB Console
+
+# Motivation
+
+Customers are currently unable to enforce their enterprise authentication policies and 
+are unable to ensure a single point of access across the product for their users.
+
+As support for CC SSO grows, it is crucial that we provide SSO for CRDB as well 
+to prevent any confusion for organization admins regarding seeing
+different authentication methods across the product. 
+
+# Background
+
+Currently, clients can authenticate themselves to the server in the following ways:
```

Comment:
This is missing session revival tokens. They were just added in v22.1 (check out https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/20211203_session_revival_token.md)

Reply:
Got it, fixed.